src/concurrency/vector_clock.rs

```diff
@@ -40,9 +45,40 @@ impl From<u32> for VectorIdx {
 /// clock vectors larger than this will be stored on the heap
 const SMALL_VECTOR: usize = 4;
 
-/// The type of the time-stamps recorded in the data-race detector
-/// set to a type of unsigned integer
-pub type VTimestamp = u32;
+/// The time-stamps recorded in the data-race detector consist of both
```

Comment:
Everything starting at consists should be on the fields. Internal docs use the flag to generate docs for private items

Reply:
This comment seems to be still open.